Doppler.ImageAnalyzer.Api/Controllers/ImageAnalyzerController.cs

```diff
@@ -52,6 +49,15 @@ private async Task<string> SaveResultsAsync(Response<List<ImageAnalysisResponse>
             var imagesAnalysis = response.Payload;
             resultId = await _imageAnalysisResultService.SaveAsync(imagesAnalysis);
         }
+        else
+        {
+            _logger.LogInformation(
```

Comment:
maybe is better log as error

Reply:
Done: https://github.com/FromDoppler/doppler-image-analyzer/pull/309/commits/9cba2a91e94d15baa15a981262a131eee027f885. 

According to what we discussed, this log info is maintained to log the errors related to the analysis itself.
Also, catch exceptions when saving the results in the db, and log them as errors.